wcfsetup/install/files/lib/http/Helper.class.php

```diff
@@ -26,6 +27,62 @@ public static function isAjaxRequest(RequestInterface $request): bool
         return $request->getHeaderLine('x-requested-with') === 'XMLHttpRequest';
     }
 
+    /**
+     * Returns the MIME type the client prefers most from the input list of
+     * $availableTypes. Checks the request's 'accept' header.
```

Comment:
I was about to point out that your `match()` statement is not exhaustive when there is a type mismatch. I think this methods is in need of an explanation of its behavior, because it actually guarantees to either return a match or the first value of `$availableTypes`.

Reply:
Okay, I've expanded the explanation of the behavior. Technically I believe the existing description already explains that the return value is an element from the input list, but it could certainly be written more explicitly.